docs/built_with.rst

```diff
@@ -0,0 +1,20 @@
+Built with aiohttp
+==================
+
+aiohttp is used to build useful libraries built on top of it,
+and there's a page dedicated to list them: :ref:`aiohttp-3rd-party`.
```

Comment:
![image](https://user-images.githubusercontent.com/1157412/28890340-29d27bce-77e0-11e7-8aff-38fe291b2144.png)

It seems this is wrong